aptos-move/framework/aptos-framework/sources/Coin.move

```diff
@@ -69,25 +79,36 @@ module AptosFramework::Coin {
     }
 
     /// Returns `true` if the type `CoinType` is a registered coin.
-    /// Returns `false` otherwise.
     public fun is_registered<CoinType>(): bool {
         let type_info = TypeInfo::type_of<CoinType>();
         let coin_address = TypeInfo::account_address(&type_info);
         exists<CoinInfo<CoinType>>(coin_address)
     }
 
+    /// Returns the name of the coin.
     public fun name<CoinType>(): ASCII::String acquires CoinInfo {
         let type_info = TypeInfo::type_of<CoinType>();
         let coin_address = TypeInfo::account_address(&type_info);
         borrow_global<CoinInfo<CoinType>>(coin_address).name
     }
 
-    public fun scaling_factor<CoinType>(): u64 acquires CoinInfo {
+    /// Returns the symbol of the coin, usually a shorter version of the name.
+    public fun symbol<CoinType>(): ASCII::String acquires CoinInfo {
         let type_info = TypeInfo::type_of<CoinType>();
         let coin_address = TypeInfo::account_address(&type_info);
-        borrow_global<CoinInfo<CoinType>>(coin_address).scaling_factor
+        borrow_global<CoinInfo<CoinType>>(coin_address).symbol
     }
 
+    /// Returns the number of decimals used to get its user representation.
+    /// For example, if `decimals` equals `2`, a balance of `505` tokens should
+    /// be displayed to a user as `5.05` (`505 / 10 ** 2`).
```

Comment:
cool, might be good to have the same notation above where this is defined

Reply:
Updated